Mathlib/Topology/UniformSpace/Defs.lean

```diff
@@ -213,83 +212,80 @@ theorem subset_iterate_compRel {s t : SetRel α α} (h : idRel ⊆ s) (n : ℕ)
   simp [IsSymmetricRel, symmetrizeRel, preimage_inter, inter_comm, ← preimage_comp]
 
 set_option linter.deprecated false in
-@[deprecated SetRel.symmetrize_subset_self (since := "2025-03-05")]
+@[deprecated SetRel.symmetrize_subset_self (since := "2025-10-17")]
 theorem symmetrizeRel_subset_self (V : SetRel α α) : symmetrizeRel V ⊆ V :=
   sep_subset _ _
 
 set_option linter.deprecated false in
-@[deprecated SetRel.symmetrize_mono (since := "2025-03-05")]
+@[deprecated SetRel.symmetrize_mono (since := "2025-10-17")]
 theorem symmetrize_mono {V W : SetRel α α} (h : V ⊆ W) : symmetrizeRel V ⊆ symmetrizeRel W :=
   inter_subset_inter h <| preimage_mono h
 
 set_option linter.deprecated false in
-@[deprecated SetRel.comm (since := "2025-03-05")]
+@[deprecated SetRel.comm (since := "2025-10-17")]
 theorem IsSymmetricRel.mk_mem_comm {V : SetRel α α} (hV : IsSymmetricRel V) {x y : α} :
     (x, y) ∈ V ↔ (y, x) ∈ V :=
   Set.ext_iff.1 hV (y, x)
 
-@[deprecated (since := "2025-03-05")]
-alias SymmetricRel.mk_mem_comm := IsSymmetricRel.mk_mem_comm
-
 set_option linter.deprecated false in
-@[deprecated SetRel.inv_eq_self (since := "2025-03-05")]
+@[deprecated SetRel.inv_eq_self (since := "2025-10-17")]
 theorem IsSymmetricRel.eq {U : SetRel α α} (hU : IsSymmetricRel U) : Prod.swap ⁻¹' U = U :=
   hU
 
-@[deprecated (since := "2025-03-05")]
-alias SymmetricRel.eq := IsSymmetricRel.eq
-
 set_option linter.deprecated false in
-@[deprecated SetRel.isSymm_inter (since := "2025-03-05")]
+@[deprecated SetRel.isSymm_inter (since := "2025-10-17")]
 theorem IsSymmetricRel.inter {U V : SetRel α α} (hU : IsSymmetricRel U) (hV : IsSymmetricRel V) :
     IsSymmetricRel (U ∩ V) := by rw [IsSymmetricRel, preimage_inter, hU.eq, hV.eq]
 
-@[deprecated (since := "2025-03-05")]
-alias SymmetricRel.inter := IsSymmetricRel.inter
-
 set_option linter.deprecated false in
-@[deprecated SetRel.isSymm_iInter (since := "2025-03-05")]
+@[deprecated SetRel.isSymm_iInter (since := "2025-10-17")]
 theorem IsSymmetricRel.iInter {U : (i : ι) → SetRel α α} (hU : ∀ i, IsSymmetricRel (U i)) :
     IsSymmetricRel (⋂ i, U i) := by
   simp_rw [IsSymmetricRel, preimage_iInter, (hU _).eq]
 
 set_option linter.deprecated false in
-@[deprecated SetRel.IsSymm.sInter (since := "2025-03-05")]
+@[deprecated SetRel.IsSymm.sInter (since := "2025-10-17")]
 lemma IsSymmetricRel.sInter {s : Set (SetRel α α)} (h : ∀ i ∈ s, IsSymmetricRel i) :
     IsSymmetricRel (⋂₀ s) := by
   rw [sInter_eq_iInter]
   exact IsSymmetricRel.iInter (by simpa)
 
 set_option linter.deprecated false in
-@[deprecated SetRel.isSymm_preimage (since := "2025-03-05")]
+@[deprecated SetRel.isSymm_id (since := "2025-10-17")]
+lemma isSymmetricRel_idRel : IsSymmetricRel (idRel : Set (α × α)) := by
+  simp [IsSymmetricRel, idRel, eq_comm]
+
+set_option linter.deprecated false in
+@[deprecated SetRel.isSymm_univ (since := "2025-10-17")]
+lemma isSymmetricRel_univ : IsSymmetricRel (Set.univ : Set (α × α)) := by
+  simp [IsSymmetricRel]
```

Comment:
As far as I can tell, these two were deleted without deprecation in #23181 ([link to file in diff](https://github.com/leanprover-community/mathlib4/pull/23181/files#diff-e45c27272452cd32af00487bc82cc30f8aba9c55c6572c1acab383141c83c60a)), so I'm adding them back here.